frontend/src/shared/components/LoadingSpinner/Styles.ts

```diff
@@ -0,0 +1,42 @@
+import { createGlobalStyle } from 'styled-components';
+
+export const LoadingSpinnerStyle = createGlobalStyle<{ color: string; size: string; borderSize: string; thickness: string }>`
```

Comment:
We should use a styled component here, not a global style.

It would look something like 

```
const LoadingSpinnerWrapper = styled.div`
  display: inline-block;
  position: relative;
  width: ${props => props.borderSize};
  height: ${props => props.borderSize};

& > div {

  box-sizing: border-box;
  display: block;
  position: absolute;
  width: ${props => props.size};
  height: ${props => props.size};
  margin: ${props => props.thickness};
  border: ${props => props.thickness} solid rgba(${props => props.theme.colors[props.color]});
  border-radius: 50%;
  animation: lds-ring 1.2s cubic-bezier(0.5, 0, 0.5, 1) infinite;
  border-color: rgba(${props => props.theme.colors[props.color]}) transparent transparent transparent;
}
...SNIP
`
  return (
      <LoadingSpinnerWrapper color={color} size={size} thickness={thickness} borderSize={borderSize}>
        <div />
        <div />
        <div />
        <div />
      </LoadingSpinnerWrapper>
  );

```


Here is a good example of using keyframes: https://styled-components.com/docs/api#keyframes

You can also take a look at the `Card/Styles.ts` for an example.